### PR TITLE
Add separate eslint packages for Typescript and Flowtype

### PR DESCRIPTION
**Pull request by ajbogh**
See PR #720 

Added separate packages for Typescript and Flowtype eslint rules.

### CODE REVIEW

**Comment by ajbogh** [comment]
This PR can be merged without incidence due to it being new code. 

#720 should not be merged until this PR is approved and merged first AND a codemod is available for internal projects. It's recommended that eslint-config-fusion use a major version bump when that time comes.

File: eslint-config-fusion-flowtype/.github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!--
```
Comment (chrisdothtml) [inline eslint-config-fusion-flowtype/.github/ISSUE_TEMPLATE.md]: Will this will work within a nested dir of a monorepo?

Comment (chrisdothtml) [reply eslint-config-fusion-flowtype/.github/ISSUE_TEMPLATE.md]: Just noticed we have these in other packages. We can leave this I guess, but will probably remove these from all packages eventually

**Comment by chrisdothtml** [comment]
!import

**Pull request closed without merging**